swagger.json

```diff
@@ -2670,7 +2670,13 @@
         "summary": "An example of New User response",
         "value": {
           "access_token": "87b11604-f554-47f8-9334-39a0d4849ab3",
-          "refresh_token": "87b11604-f554-47f8-9334-39a0d4849ab3"
+          "refresh_token": "87b11604-f554-47f8-9334-39a0d4849ab3",
```

Comment:
Please add user model to LoginResponse [model](https://github.com/Code-the-Dream-School/ii-practicum-team-2-back/blob/46148054f0198bac24ad5fe060faec11c8c0638e/swagger.json#L1935) as well.

Could be reused from [Register](https://github.com/Code-the-Dream-School/ii-practicum-team-2-back/blob/46148054f0198bac24ad5fe060faec11c8c0638e/swagger.json#L1910).